packages/flutter/lib/src/material/material.dart

```diff
@@ -80,24 +80,12 @@ const Map<MaterialType, BorderRadius?> kMaterialEdges = <MaterialType, BorderRad
 /// An interface for creating [InkSplash]s and [InkHighlight]s on a [Material].
 ///
 /// Typically obtained via [Material.of].
-abstract class MaterialInkController {
-  /// The color of the material.
-  Color? get color;
-
-  /// The ticker provider used by the controller.
-  ///
-  /// Ink features that are added to this controller with [addInkFeature] should
-  /// use this vsync to drive their animations.
-  TickerProvider get vsync;
-
-  /// Add an [InkFeature], such as an [InkSplash] or an [InkHighlight].
-  ///
-  /// The ink feature will paint as part of this controller.
-  void addInkFeature(InkFeature feature);
-
-  /// Notifies the controller that one of its ink features needs to repaint.
-  void markNeedsPaint();
-}
+//
+// TODO(nate-thegrate): deprecate Material-specific symbols once Splash
+// fully replaces them.
+// The Material widget will be refactored to use a SplashBox in its build method.
+// https://flutter.dev/go/layered-material-widgets
```

Comment:
An issue will be preferred, since we can track the progress of the migration/project in the issue. You can link the doc in the issue.